service/builder/exporters_builder.go

```diff
@@ -193,22 +196,23 @@ func (eb *ExportersBuilder) buildExporter(
 		return nil, fmt.Errorf("exporter factory not found for type: %s", config.Type())
 	}
 
-	exporter := &builtExporter{}
+	exporter := &builtExporter{
+		logger: logger,
```

Comment:
Looks like we initialize `builtExporter.logger` with the same logger, so it does not have any custom fields that could justify the need to have it as a field in `builtExporter`. Again, probably I am missing something.

Reply:
https://github.com/open-telemetry/opentelemetry-collector/pull/829/files/1bedf14b1dfb74f4176d1f2991042944e5925597#diff-b41649f7617e90587ef5c12dc7cfe3bdR131

https://github.com/open-telemetry/opentelemetry-collector/pull/829/files/1bedf14b1dfb74f4176d1f2991042944e5925597#diff-b41649f7617e90587ef5c12dc7cfe3bdR144

The logger we put here has kind/type/name associated with it.